### DISCUSSION TITLE
set font-family to css variable

### DISCUSSION THREAD

**Discussion by ian-hw** [tailwindlabs/tailwindcss#11991] [Help]
Is there a way to set font-family to a CSS variable? My styles need to reference a CSS variable that is located in another stylesheet. `font-[var(--some-variable)]` converts to `font-weight: var(--some-variable);`.

**Comment by wongjn** [comment]
You could consider ["type casting" the value](https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities) like one of:
- `font-[lookup:--some-variable]`
- `font-[generic-name:--some-variable]`
- `font-[family-name:--some-variable]`

**Reply by ian-hw** [reply]
This worked - thanks again! I appreciate you watching this and discord lol

**Comment by victormarcel09** [comment]
You store the font name in a CSS variable and apply it to the font-family property where needed. If you need to reference the variable from another stylesheet, make sure to import or include the stylesheet where the variable is defined. [Follow My Health Sharp](https://www.followmy-health.com/)
